pkg/cmd/container/inspect.go

```diff
@@ -47,12 +50,45 @@ func Inspect(ctx context.Context, client *containerd.Client, containers []string
 			log.L.Error(formatErr)
 		}
 	}
+
 	return err
 }
 
 type containerInspector struct {
-	mode    string
-	entries []interface{}
+	mode        string
+	size        bool
+	snapshotter snapshots.Snapshotter
+	entries     []interface{}
+}
+
+func resourceTotal(ctx context.Context, snapshotter snapshots.Snapshotter, resourceID string) (snapshots.Usage, snapshots.Usage, error) {
```

Comment:
Could you add a comment to explain the returned values?

Reply:
Yes.